.pre-commit-config.yaml

```diff
@@ -12,12 +12,12 @@ repos:
       - id: typos
         # https://github.com/crate-ci/typos/issues/347
         pass_filenames: false
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.0.0-alpha.4
-    hooks:
-      - id: prettier
-        additional_dependencies:
-          - prettier
+  # - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
Is there an eslint pre-commit? That way it gets auto-fixed and enforced...

(no huuge stress if not, particularly since you're the main contributor atm)

Reply:
don't know because I don't use pre-commit bot. maybe someone else can look into it.

I do like that it fixes my typos. I do that a lot! :)